src/constants/tokens.ts

```diff
@@ -81,7 +81,20 @@ export const USDC_POLYGON_MUMBAI = new Token(
   'USDC',
   'USD//C'
 )
-
+export const PORTAL_USDC_CELO = new Token(
+  SupportedChainId.CELO,
+  '0x37f750B7cC259A2f741AF45294f6a16572CF5cAd',
+  6,
+  'USDCet',
```

Comment:
This is not included in the token list. Should it be one of the common base tokens (always displayed on the top of the selector)?